cost/budget_report_alerts/README.md

```diff
@@ -19,12 +19,14 @@ The [Provider-Specific Credentials](https://docs.flexera.com/flexera/EN/Automati
 - Forecasted Spend budget alerts will raise an incident when the target's run-rate is on track to exceed the budget threshold
 - Data can be grouped by Dimensions.
 - The policy allows the customer to include or exclude unbudgeted spend
+- The policy allows multiple filter group values to scope specific dimension value like Service=AmazonEC2
```

Comment:
This should be reworked to use proper grammar. We also don't need to provide examples, since the parameter section of the README will contain that information.

Example of what might work here:

The policy allows budgets to be tracked for specific dimensions and values.